tests/test_null_byte_insertion.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2021 The Matrix.org Foundation C.I.C.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import synapse.rest.admin
+from synapse.rest.client import account, login, register, room
+
+from tests.unittest import HomeserverTestCase
+
+
+class NullByteInsertionTest(HomeserverTestCase):
+    servlets = [
+        synapse.rest.admin.register_servlets_for_client_rest_resource,
+        login.register_servlets,
+        account.register_servlets,
+        register.register_servlets,
+        room.register_servlets,
+    ]
+
+    def setUp(self):
+        super().setUp()
+
```

Comment:
A missing method automatically goes up to the super-class, so no reason to include it.

```suggestion
    def setUp(self):
        super().setUp()

```

Reply:
(uhh, the suggested change is one of adding a newline?)